compiler/control/CompileMethod.cpp

```diff
@@ -312,6 +312,10 @@ compileMethodFromDetails(
             compiler.setDebug(createDebugObject(&compiler));
             }
 
+#ifdef VERIFY_IL
```

Comment:
Having an option depend on a build flag or configuration is not unprecedented (its not necessarily ideal, but our options framework needs a lot of re-work).

I think a runtime option is ideal, but I'm not going to press the issue for this PR.  However, if this is only dependent on the Test compiler at the moment and will be for some time, then I think using the `#ifdef` `TEST_PROJECT_SPECIFIC` is a better choice than `VERIFY_IL`.  Is `VERIFY_IL` used anywhere else?